docker/lr-jupyter/Dockerfile

```diff
@@ -0,0 +1,122 @@
+FROM jupyter/datascience-notebook
```

Comment:
I am sure you have a reason, but can you explain why the official image isn't good enough?

Reply:
I think at the time this seemed like the correct docker to start from.  Jupyter publishes several versions of the notebook container, so this is actually an official image (maybe just one that fewer people use).